_includes/modern-appendix.md

```diff
@@ -843,6 +851,31 @@ The text used in the last param of this message varies wildly.
 
 Indicates that there was a problem with a mode parameter. Replaces various implementation-specific mode-specific numerics.
 
+{% numericheader RPL_HELPSTART %}
+
+    "<client> <subject> :<first line of help section>"
+
+Indicates the start of a reply to a {% command HELP %} command.
+The text used in the last param of this message may vary, and SHOULD be displayed as-is by IRC clients to their users; possibly emphasized as a the title of the help section.
+
+The `<subject>` MUST be the one requested by the client, but may be casefolded; unless it would be an invalid parameter, in which case it MUST be `*`.
+
+{% numericheader RPL_HELPTXT %}
+
+    "<client> <subject> :<line of help text>"
+
+Returns a line of {% command HELP %} text to the client. Lines MAY be wrapped to a certain line length by the server. Note that the final line MUST be a {% numeric RPL_ENDOFHELP %} numeric.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
+
+{% numericheader RPL_ENDOFHELP %}
+
+    "<client> <subject> :<last line of help text>"
+
+Returns the final {% command HELP %} line to the client.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
```

Comment:
```suggestion
The `<subject>` MUST be the one requested by the client, but may be casefolded; unless it would be an invalid parameter, in which case it MUST be `*`.

{% numericheader RPL_ENDOFHELP %}

    "<client> <subject> :<last line of help text>"

Returns the final {% command HELP %} line to the client.

The `<subject>` MUST be the one requested by the client, but may be casefolded; unless it would be an invalid parameter, in which case it MUST be `*`.
```